docs/package.json

```diff
@@ -23,7 +23,7 @@
     "prod:deploy": "node scripts/deploy/aws.js",
     "prettier-all": "npx prettier --write '{src,static-config-helpers,static-config-parts,static.config,scripts,lib}/**/*.{js,jsx}'",
```

Comment:
For a future ticket, let's get rid of all `npx`. It's much better and faster to use our actual deps.